runbotics/runbotics-desktop/src/config/BackgroundPageApiRequestHandler.ts

```diff
@@ -56,21 +56,34 @@ export class BackgroundPageApiRequestHandler extends StatelessActionHandler impl
             }
         }
         let response;
-        const method = input.method ? input.method : 'get';
+        const method = input.method ? input.method : 'GET';
         try {
             switch (method) {
-                case 'get':
+                case 'GET':
                     response = await Axios.get(compiled, { headers: input.headers });
                     break;
-                case 'post':
+                case 'DELETE':
+                    response = await Axios.delete(compiled, JSON.parse(input.body));
+                    break;
+                case 'PATCH':
+                    response = await Axios.patch(compiled, JSON.parse(input.body), { headers: input.headers });
+                    break;
+                case 'PUT':
+                    response = await Axios.put(compiled, JSON.parse(input.body), { headers: input.headers });
+                    break;
+                case 'POST':
                     response = await Axios.post(compiled, JSON.parse(input.body), { headers: input.headers });
                     break;
                 default:
                     response = await Axios.get(compiled, { headers: input.headers });
                     break;
             }
         } catch (e) {
-            throw e;
+            this.logger.log(e); 
```

Comment:
I dont think that we always get the response we always should throw an error. This if statement should be deleted imo